src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -36,8 +36,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         FeedbackResponseAttributes response = logic.getFeedbackResponse(feedbackResponseId);
         Assumption.assertNotNull(response);
 
-        verifyAccessibleForInstructorToFeedbackResponseComment(feedbackResponseCommentId,
-                                                               instructor, session, response);
+        FeedbackResponseCommentAttributes frc =
+                logic.getFeedbackResponseComment(Long.parseLong(feedbackResponseCommentId));
+        if (frc == null) {
+            Assumption.fail("FeedbackResponseComment should not be null");
+        } else {
```

Comment:
@chowyb 
Isn't there something like `Assumption.assertNotNull(frc)` that we can use here instead?